package.json

```diff
@@ -2,26 +2,23 @@
   "name": "CRUD.js",
   "version": "1.0.0",
   "description": "A CRUD JavaScript class.",
-  "main": "CRUD.js",
+  "main": "dist/CRUD.min.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/HugoGiraudel/CRUD.js.git"
   },
   "devDependencies": {
-    "grunt": "~0.4.1",
-    "grunt-cli": "~0.1.9",
-    "grunt-contrib-concat": "~0.3.0",
-    "grunt-contrib-uglify": "~0.2.0",
-    "grunt-contrib-watch": "~0.5.2",
-    "grunt-contrib-clean": "~0.5.0",
-    "grunt-contrib-jasmine": "~0.5.2",
-    "grunt-contrib-jshint": "~0.8",
-    "grunt-contrib-connect": "~0.5.0",
-    "grunt-template-jasmine-requirejs": "~0.1.6",
-    "grunt-template-jasmine-istanbul": "~0.2.5",
-    "open": "~0.0.4",
-    "argparse": "~0.1.15",
-    "underscore": "~1.5.2",
-    "glob": "~3.2.6"
+    "babelify": "^6.3.0",
+    "http-server": "^0.8.4",
+    "npm-watch": "0.0.1",
+    "uglify-js": "^2.5.0"
+  },
+  "watch": {
+    "build": "src/*.js"
+  },
+  "scripts": {
+    "build": "browserify src/CRUD.js -t babelify --outfile dist/CRUD.js && uglifyjs dist/CRUD.js --compress --mangle --output dist/CRUD.min.js",
+    "watch": "node_modules/http-server/bin/http-server ./ & npm-watch",
```

Comment:
`npm run` will set `node_module/.bin` in `PATH`, which contains `node_modules/http-server/bin/http-server`:

```sh
http-server ./ & npm-watch
```